### ISSUE TITLE
Enable CI builds and tests for ESP32-C2, ESP32-S2, ESP32-H2, ESP32-C6 (IEC-12)

### ISSUE THREAD

**Issue by igrr** [espressif/idf-extra-components#150]
At least for ESP32-S2 and ESP32-C2 this is probably dependent on #149, as otherwise the flash/RAM size isn't sufficient for the test app.

**Comment by igrr** [comment]
Closing in favor of https://github.com/espressif/idf-extra-components/issues/387.
